chat-widget/samples/javascript-sample/SampleWidget.js

```diff
@@ -12,123 +10,140 @@ import { version as chatComponentVersion } from "@microsoft/omnichannel-chat-com
 import { version as chatSdkVersion } from "@microsoft/omnichannel-chat-sdk/package.json";
 import { version as chatWidgetVersion } from "../../package.json";
 import { getCustomizationJson } from "./getCustomizationJson";
-import { memoryDataStore } from "./Common/MemoryDataStore";
 import getMockChatSDKIfApplicable from "./getMockChatSDKIfApplicable";
+import { memoryDataStore } from "./Common/MemoryDataStore";
 
 let liveChatWidgetProps;
 
-const main = async () => {
-
+async function fetchAuthTokenViaNetlify() {
+    const url = "https://omnichannel-auth-server.netlify.app/.netlify/functions/api/login";
```

Comment:
Do we want to leave this in the sample widget? Also, is this app still up and running? Is it an azure function or something that returns the auth token? If so, can I repurpose this for my own testing? 😉